problem2.py

```diff
@@ -0,0 +1,21 @@
+#brute force Solution
+#time Comp- O(nlogn) Sorting array Not very sure correct if i am wrong
+#space-O(n) extra space needed for array ans
+class Solution:
```

Comment:
Sorting according to start times is correct, after that to keep track of meetings are being completed use min heap for the end times